assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.xml

```diff
@@ -291,22 +585,161 @@
+            <param name="referencedColumns">Collection with all columns that are used to trigger on.</param>
+            <exception cref="T:System.ArgumentOutOfRangeException">One of the passed parameters contains a value that was not expected.</exception>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ValidateRateColumns(System.Collections.Concurrent.ConcurrentDictionary{System.Int32,System.Int32})">
+            <summary>
+            Validates the rate columns.
+            </summary>
+            <param name="rateColumns">Collection with rate columns.</param>
+            <exception cref="T:System.ArgumentOutOfRangeException">Exception thrown when a rate column is not a customized type or if the data grid does not contain the rate column.</exception>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ValidateReferencedColumns(System.Collections.Generic.HashSet{System.Int32})">
+            <summary>
+            Validates the referenced columns.
+            </summary>
+            <param name="referencedColumns">Collection with referenced columns.</param>
+            <exception cref="T:System.ArgumentOutOfRangeException">Exception thrown when a referenced column is not a customized type or if the data grid does not contain the referenced column.</exception>
+            <exception cref="T:System.ArgumentException">Exception thrown when no function is assigned to be executed when the referenced columns change value.</exception>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.ValidateReferencedColumn(Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.IDataMinerConnectorDataGridColumn)">
+            <summary>
+            Validates a destination of a referenced column.
+            </summary>
+            <param name="destinationColumn">Destination column to be verified</param>
+            <exception cref="T:System.ArgumentOutOfRangeException">Exception thrown when the source of a destination column is not a poll type or if the data grid does not contain the source of a destination column.</exception>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.SealColumns">
+            <summary>
+            Seals the columns after initialization, so e.g. column types can no longer be changed during runtime.
+            </summary>
+        </member>
+        <member name="T:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Protocol.DataMapper.DataMinerConnectorDataGrid.TokenPropertyInvestigationResult">
+            <summary>
+            Class that is being used while iterating over the properties what the found key and token is.
```

Comment:
I'm not sure how to understand this. Do you mean the class that is used to iterate the found key and token over the properties?